lms/djangoapps/instructor/views/api.py

```diff
@@ -1047,6 +1057,7 @@ def re_validate_invoice(obj_invoice):
 @ensure_csrf_cookie
 @cache_control(no_cache=True, no_store=True, must_revalidate=True)
 @require_level('staff')
+@sudo_required
 def get_students_features(request, course_id, csv=False):  # pylint: disable=redefined-outer-name
```

Comment:
@dan-f Yeah I know that but we need to sure about that when someone do an ajax request without sudo privileges he should redirected to sudo page.

As we are giving user a sudo privileges when he/she visits instructor dashboard so it will not hurt us.